wiki/Community/Discord_servers/en.md

```diff
@@ -1,9 +1,89 @@
+| [osu! Medal Hunters](https://discord.gg/osumedalhunters) | [MegaMix](https://osu.ppy.sh/users/18152711) | **osu! Medal Hunters** is the main Discord server used by the medal hunting community, where people discuss [medals](/wiki/Medals) and [medal solutions](/wiki/Medals/Unlock_requirements). |
+
+## Development
+
+These servers focus on providing and developing tools and systems for the game. While the official development of osu! is done in the official community Discord server, certain projects require their own workspaces before proposing and merging their work in game.
+
+**The following servers are not officially affiliated with the game.** To follow up with official updates, join the [osu! Discord server](https://discord.gg/ppy) instead.
+
+| Name | Owner | Description |
+| :-- | :-- | :-- |
+| [Performance Points](https://discord.gg/aqPCnXu) | - | - |
+| [osu!catch dev server](https://discord.gg/YEJBENvFzN) | - | - |
+| [o!m SR/PP Rework Hub](https://discord.gg/GFCNNg8bwk) | [Quenlla](https://osu.ppy.sh/users/4725379) | The **osu!mania SR/PP Rework Hub** aims to offer a place to discuss active developments from the community to improve the current difficulty systems used by osu!mania. Users can view and discuss the different ideas developed by their peers before they get officially proposed. |
+
+## Mapping and Modding
+
+These servers aim to provide safe places for both mappers and modders alike. Users can expect to find a trove of both mapping and modding knowledge in these for their respective modes!
+
+| Name | Owner | Description |
+| :-- | :-- | :-- |
+| [Mapset Management Server](https://discord.gg/8BquKaS) | [Kyuunex](https://osu.ppy.sh/users/9236044) | The **Mapset Management Server** is all about modding and collaboration. Whether you are looking for mods and GDs on your own maps, or other maps to mod and GD or collaborate, you're in the right place! |
+| [Cozy Mappers' Café](https://discord.gg/uDjwjaZ ) | - | - |
+
+### Hubs
+
+The **Modding & Mapping Hubs** are community Discord servers designated for all current and aspiring mappers and modders. Users are able to find many channels that help them grow as a mapper, or interact with the general mapping community. Each mode boasts of its own hub, and each one is ran by different community members.
+
+| Name | Owner |
+| :-- | :-- |
+| [osu!catch Modding & Mapping Hub](https://discord.gg/ZuxFc4q) | [Ascendance](https://osu.ppy.sh/users/2931883) |
+| [osu!taiko Modding & Mapping Hub](https://discord.gg/yRjvvyZ) | [Hivie](https://osu.ppy.sh/users/14102976) |
+| [osu!mania Modding & Mapping Hub](https://discord.gg/FqbDdYN) | [Maxus](https://osu.ppy.sh/users/4335785) |
+
+## Skinning
+
+(Better to ask Rock Roller on this. They'll probably do better than me)
+
+| Name | Owner | Description |
+| :-- | :-- | :-- |
+| [skinship](https://discord.gg/AZ79xJb) | - | - |
```

Comment:
```suggestion
| [skinship](https://discord.skinship.xyz/) | [RockRoller](https://osu.ppy.sh/users/8388854) | **skinship** is a place to discuss anything involving skinning, while also hosting projects such as [Skinning Contests](/wiki/en/Contests/Skinning_Contest) and the [Completed Skins Compendium](https://compendium.skinship.xyz/).  |
```

- This invite link is used in other articles and newspost
- Not sure how to word that request should *not* be send to me, but if possible to the [skinship bot account](https://osu.ppy.sh/users/32231941) instead
- No clue what to put as the description really, but guess this is better than what we use on the discovery page for skinship (`all about osu! skinning`)

Reply:
We can add an addendum in the description to forward users to the bot account, while keeping the owner there if there are any inquiries about the server itself.